app/routes/users.js

```diff
@@ -4,22 +4,37 @@ const userService = require('../services').userService;
 
 /* GET users listing. */
 router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
+  return req.sessionID ? res.redirect('/#!/signin') : res.redirect('/#!/signup');
```

Comment:
This logic should be contained inside a custom middleware since this will likely be reused for multiple routes.